### PR TITLE
[CMS-1096] De-emphasize terminus 1.x and 2.x docs.

### PR DESCRIPTION
**Pull request by kporras07**
Closes CMS-1096

## Summary

Several pages changed, mostly under:
**[Terminus guide](https://pantheon.io/docs/terminus)** - De-emphasizes and remove terminus 1.x and 2.x references in docs.

**Release**:
- [X] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `https://pantheon.io/docs/terminus-2-0` => `https://pantheon.io/docs/terminus/terminus-3-0` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/terminus-2-0.md
Comment (kporras07) [inline source/content/terminus-2-0.md:9]: Actually, I'd love to remove this page and redirect to terminus-3-0 (https://pantheon.io/docs/terminus/terminus-3-0). Let me know if this is possible and how could I help to make it happen

Comment (whitneymeredith) [reply source/content/terminus-2-0.md:9]: Agreed!

**Review by whitneymeredith — APPROVED** [review]

**Comment by whitneymeredith** [comment]
@kporras07 LGTM. I'll wait for the @pantheon-systems/cms-ecosystem to sign off on this.

File: source/content/terminus/07-create.md
Comment (namespacebrian) [inline source/content/terminus/07-create.md:128]: Should this be 7.4? 😉

File: source/content/supported-terminus.md
Comment (kporras07) [reply source/content/supported-terminus.md:2]: A redirect should definitely be created. @whitneymeredith are you taking care of this after merging?

File: source/content/terminus/07-create.md
Comment (kporras07) [reply source/content/terminus/07-create.md:128]: Great catch! Thanks!

File: source/content/supported-terminus.md
Comment (whitneymeredith) [reply source/content/supported-terminus.md:2]: That's our typical redirect workflow.

**Review by namespacebrian — APPROVED** [review]

**Comment by kporras07** [comment]
@whitneymeredith all yours. It has now been signed off by cms eco team :)

**Pull request merged**
